packages/runtime/src/client/tests/client.get-component-snapshot.test.ts

```diff
@@ -93,19 +93,18 @@ describe('getComponentSnapshot using v1 element tree endpoint', () => {
             - this is the step that we're really testing here - we want the logic in getComponentSnapshot to execute this subsequent request
         (3) graphql query for introspection
     */
+      const httpHandler = jest.fn(({ request }) => {
```

Comment:
Here and below, follow MSW recommendations to use explicit request URL parameters parsing: https://mswjs.io/docs/http/intercepting-requests/query-parameters.

Wrapping the handler in `jest.fn` so we can assert on the number of calls and/or parameters.